processing/src/main/java/io/druid/query/QueryToolChest.java

```diff
@@ -195,4 +201,13 @@
   {
     return segments;
   }
+
+  /**
+   * Creates Ordering for merging partial results
+   */
+  public <X extends Comparable> Ordering<X> getOrdering(boolean descending)
```

Comment:
I'm still at the top of the review, so not sure where this method is being called, yet.  But, in general, the merge functionality is entirely owned by the concrete implementation of this interface, so I'm not sure this method needs to be added to the interface.  Perhaps when I see where it is being called, it will make more sense.